.gitignore

```diff
@@ -4,7 +4,7 @@
 .eslintcache
 
 # Generated
-build/
+/build/
```

Comment:
Is this change necessary? It works fine for me without the slash at the beginning.

Reply:
I've made this adjustment to the .gitignore file because it was ignoring all files in any 'build' folder. This change was necessary after adding the 'build' folder to the content folder for the build page addition. By inserting a vertical bar, I ensured that only the 'build' folder in the root directory is ignored, not the ones in other locations.

Reply:
I see, thanks for the explanation!